11-query/04-delete-query.md

```diff
@@ -46,39 +48,81 @@ delete $emp;
+Attributes are normally owned by other concepts (ie. `$x has attribute $a`). This means we can both delete an attribute
+ownership, as in the next section, or delete the instance itself following the above style of using `isa` in the delete block.
 
-To delete only the association that a thing has with an attribute, we use the `via` keyword to capture and delete the relation between the owner and the owned attribute - NOT the instance of the attribute type itself, as doing so disowns the instance from any other instance that may have owned it.
+## Delete Attribute Ownerships
+We can remove the ownership of an attribute by another concept in the same way it is inserted or queried: using the `has` clause.
+
+Note that attributes with the same value are shared among their owners, so usually shouldn't delete the attribute directly. 
+It's important that one understands thoroughly how [attributes are defined](../09-schema/01-concepts.md#define-an-attribute).
 
 <div class="tabs dark">
 
 [tab:Graql]
 ```graql
-match $t isa travel, has start-date 2013-12-22 via $r; delete $r;
+match $t isa travel, has start-date $st; $d 2013-12-22; delete $t has $st;
 ```
 [tab:end]
 
 [tab:Java]
 ```java
 GraqlDelete query = Graql.match(
-  var("t").isa("travel").has("start-date", var("st"), var("r")),
+  var("t").isa("travel").has("start-date", var("st")),
   var("st").val(LocalDate.of(2013, 12, 22).atStartOfDay())
-).delete("r");
+).delete(var("t").has(var("st")));
 ```
 [tab:end]
 </div>
 
-This looks for a `travel` that owns the attribute `start-date` with the value of `2013-12-22`, captures the association between the attribute and the owner as the variable `$r` and finally deletes `$r`. This ensures that the attribute instance of type `start-date` and value `2013-12-22` remains associated with any other instance that may own it.
+This looks for a `travel` that owns the attribute `start-date` with the value of `2013-12-22` in the `match` clause. 
+We then disassociate the `travel` instance `$t` from the `start-date` attribute `$st` with the `delete $t has $st` clause.
+
+This will _not_ delete the entire instance of type `start-date` and value `2013-12-22` - it remains associated with any other instance that may own it.
+
+If we had instead written the query as `match $t isa travel, has start-date $st;  $st == 2013-12-22"; delete $st isa start-date;`, 
+we would have deleted the instance of `start-date` with value `2013-12-22` and its association with all other concept types that previously owned it.
+
+## Delete Role Players from Relations
+
+In Grakn, existing relations can be extended with new role players, or shrunk by removing role players. If we had an employment relation,
+and the employer company merged with another and needed to be reassigned to the new company, we would first have to remove
+the prior company from the employment relation.
```

Comment:
this part: `and the employer company merged with another and needed to be reassigned to the new company` 
doesn't read so well.

Reply:
ok, new version!